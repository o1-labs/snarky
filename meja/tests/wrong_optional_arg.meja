let a = fun (~a) => {
  a;
};

let b = fun () => { a(?a = Some(15)); };
