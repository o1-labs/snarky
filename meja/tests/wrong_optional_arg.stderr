File "tests/wrong_optional_arg.meja", line 5, characters 32-34:
Error: Incompatible types option('a) and int:
       Cannot unify option('a) and int
